source/support/fob.sqf

```diff
@@ -1,14 +1,30 @@
+/*
+    File: fob.sqf
+
+    Author: Kibot
+
+    Description:
+        Searches for a position and setups up Forward Operating Base
+
+    Parameter(s):
+        _this select 0 - OBJECT - A non-nil object with a physical location.
+
+    Usage:
+        _scriptHandle = [player] execVM 'fob.sqf';
```

Comment:
Is there any need to store the script handle?
You really should pre-compile this into a variable and then `call` or `spawn` it if it's used more than once.